base/markdown/Julia/interp.jl

```diff
@@ -1,8 +1,8 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-function Base.parse(stream::IO; greedy::Bool = true, raise::Bool = true)
+function Meta.parse(stream::IO; greedy::Bool = true, raise::Bool = true)
```

Comment:
This method feels a bit out of place.

Reply:
https://github.com/JuliaLang/julia/issues/22924

Reply:
I'll try just making it private then.